app/styles/theme.scss

```diff
@@ -16,6 +16,7 @@ $Theme-opacity-white: 0.7;
 $Theme-opacity-black: 0.6;
 
 $Theme-break-mobile: 699px;
+$Theme-break-mobile-nav: 1167px;
```

Comment:
As we discussed in person, this makes more sense to move to the `_nav.scss` file and rename it to something like `$nav-break-mobile`